cp/custom-cp.mk

```diff
@@ -3,4 +3,9 @@
 
 #ZMQ_COMM flag is set for direct ZMQ communication, unset for direct UDP communication
 #ZMQ communication is enabled by default
-#CFLAGS += -DZMQ_COMM
+CFLAGS += -DZMQ_COMM
```

Comment:
If possible, We should have a common custom.mk which is referred by cp/dp.

Reply:
Okay.

Reply:
Or should interface be compiled separately with it's own makefile? Have that as dependency for cp and dp?

Reply:
Good idea. Will work on this.

Reply:
For the time being, I have created a common file with the name of `vars.mk` that is shared by both CP and DP.